aspnetcore/migration/1x-to-2x/index.md

```diff
@@ -99,6 +99,27 @@ The adoption of this new 2.0 pattern is highly recommended and is required for p
 Unable to create an object of type '<Context>'. Add an implementation of 'IDesignTimeDbContextFactory<Context>' to the project, or see https://go.microsoft.com/fwlink/?linkid=851728 for additional patterns supported at design time.
 ```
 
+<a name="add-modify-configuration"></a>
+
+## Add configuration providers
+In 1.x projects, adding configuration providers to an app was accomplished during the middleware registration process. The steps involved creating an instance of `ConfigurationBuilder`, loading applicable providers (environment variables, app settings, etc.), and initializing a member of `IConfigurationRoot`.
```

Comment:
>  adding configuration providers to an app was accomplished during the middleware registration process

adding configuration providers to an app was accomplished via the Startup constructor.